massa-client/src/tests/scenarios.rs

```diff
@@ -1,10 +1,30 @@
+use super::tools;
+use assert_cmd::Command;
 use serial_test::serial;
+use std::thread;
 
 #[tokio::test]
 #[serial]
 async fn test_if_exit_gracefully() {
-    use assert_cmd::Command;
+    let mut cmd = Command::cargo_bin("massa-client").unwrap();
+    cmd.arg("exit").assert().success();
+}
 
+const CONFIG_PATH: &str = "../massa-node/base_config/testnet.toml";
+
+#[tokio::test]
+#[serial]
+async fn test_if_wallet_info() {
+    tools::update_genesis_timestamp(CONFIG_PATH);
+    let massa_node_thread_handle = thread::spawn(|| {
+        Command::cargo_bin("massa-node")
+            .unwrap()
+            .env("MASSA_CONFIG_PATH", CONFIG_PATH)
+            .unwrap();
+    });
     let mut cmd = Command::cargo_bin("massa-client").unwrap();
+    thread::sleep(std::time::Duration::from_secs(5));
```

Comment:
Since this is in a tokio test, it would be better to use the async sleep api if possible.

Reply:
yeap!